pkgs/development/libraries/imgui/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "imgui";
+  version = "1.85";
+
+  src = fetchFromGitHub {
+    owner = "ocornut";
+    repo = pname;
```

Comment:
I'm not sure if I would go for recursion just for this.

Reply:
Fair enough, changed it